profile/README.md

```diff
@@ -0,0 +1,14 @@
+# CoreOS
+
+## The container optimized OS
+
+We are building minimal Operating Systems (RHEL CoreOS, Fedora CoreOS) focused on running containerized workflows, with a focus on security, scalability and automatic updates. We are using [OCI containers as a transport and delivery mechanism for operating system content](https://containers.github.io/bootable/) by using [bootc](https://github.com/bootc-dev/bootc)
```

Comment:
Can we split that on two lines ? 
Also, we are not using bootc (yet). All the container native stuff still happen in rpm-ostree